app/res/values/strings.xml

```diff
@@ -352,4 +352,17 @@ Some images in CommCare are graciously used under Creative Commons Licensing fro
     <string name="sign_button" cc:translatable="true">Gather Signature</string>
     <string name="markup_image" cc:translatable="true">Markup Image</string>
     <string name="draw_image" cc:translatable="true">Sketch Image</string>
+
+    <!--Incorrect Target Screen-->
+    <string name="target_mismatch_install_help" cc:translatable="true">If you have any issues installing, please contact your Helpdesk</string>
+    <string name="target_mismatch_lts_required" cc:translatable="true">App Requires CommCare LTS</string>
+    <string name="target_mismatch_lts_required_install" cc:translatable="true">Install CommCare LTS</string>
+    <string name="target_mismatch_lts_open" cc:translatable="true">Open CommCare LTS</string>
+    <string name="target_mismatch_lts_required_info" cc:translatable="true">In order to install CommCare LTS, Please click on install button below. Then try installing your app again.</string>
```

Comment:
Slight modification suggestion (and the same for the parallel non-LTS string):
`Follow the link below to install CommCare LTS from the Play Store, and then try installing your app again.`